### ISSUE TITLE
elevation map service save only the visible map not complete map

### ISSUE THREAD

**Issue by Thedush** [ANYbotics/elevation_mapping#183]
the elevation map was generating it called the tigger_fusion and start generating map then i saved the map
it got saved but saves only the visible region

**Comment by maximilianwulf** [comment]
No, this package does not provide a planner. But you will easily find them online.

**Comment by Thedush** [comment]
i have searched a lot for the planner i couldnt find one. if you know any planner can you suggest me one
